marklogic-langchain4j/src/main/java/com/marklogic/langchain4j/embedding/DocumentAndChunks.java

```diff
@@ -64,7 +63,7 @@ public String getTemporalDocumentURI() {
     }
 
     @Override
-    public int compareTo(@NotNull DocumentWriteOperation o) {
+    public int compareTo(DocumentWriteOperation o) {
```

Comment:
Just curious as to why you removed the @NotNull.

Reply:
It was an Intellij annotation that had been added, unbeknownst to me.